hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/systemcontracts/CallAttemptHelpers.java

```diff
@@ -126,4 +126,28 @@ public static HasCallAttempt prepareHasAttemptWithSelector(
                 List.of(translator),
                 false);
     }
+
+    public static HtsCallAttempt prepareHtsAttemptWithSelectorForRedirectWithConfig(
```

Comment:
looks like this is the same code as in the Cancel Airdrop pr?  I guess you need it for testing.

Reply:
Yes, exactly. Both system contracts are very similar.